lib/models/users-model.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Model = require('./model.js');
+const schema = require('./users-schema.js');
+
+class Users extends Model {
+  constructor() {
+    super(schema);
+  }
+
+  // TODO JSDocs Comment
+  /**
+   * @function authenticate
+   * @param  {} creds 
+   * @returns 
```

Comment:
You can be more specific with the types like `@param {object} creds` so it is clear for whoever is reading your code.  What will this authenticate function return?